app/blueprints/contribution_blueprint.rb

```diff
@@ -3,9 +3,12 @@
 class ContributionBlueprint < Blueprinter::Base
   identifier :id
   association :categories_for_tags, name: :category_tags, blueprint: DefaultBlueprint
-  association :service_area, blueprint: ServiceAreaBlueprint, view: :with_location
+  association :service_area, blueprint: ServiceAreaBlueprint, view: :with_location do |contribution|
+    contribution.service_areas.first
```

Comment:
- [ ] TODO: @exbinary suggested that we could change this blueprint to output multiple service areas and the Vue code to display them. 
 
(Otherwise, we might have a community resource in areas `a` & `b` and someone filters by area `b` and is surprised to see a resource that shows up with only `a` listed as its area)

Reply:
Done!